### PR TITLE
Create pipelines to build iOS Debug Proxy for Windows and MacOS

### PR DESCRIPTION
**Pull request by scout719**

### CODE REVIEW

File: MSYS_install/install.ps1 [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+function bash($command) {
+    Write-Host $command -NoNewline
+    cmd /c start /wait C:\msys64\usr\bin\sh.exe --login -c $command
+    Write-Host " - OK" -ForegroundColor Green
+}
+
+[Environment]::SetEnvironmentVariable("MSYS2_PATH_TYPE", "inherit", "Machine")
+
+# install latest pacman
+#bash 'pacman -Sy --noconfirm pacman pacman-mirrors'
```
Comment (AnaIsabelSousa) [inline MSYS_install/install.ps1]: Can we remove the lines with code that are commented? The same is happening below, I'm guessing these were experiments from Constantine?

File: ios_debug_proxy.yml [hunk 2]
```diff
@@ -0,0 +1,179 @@
+#ios_debug_proxy pipeline for getting libraries and binary
+
+trigger: none
```
Comment (AnaIsabelSousa) [inline ios_debug_proxy.yml]: Shouldn't we have an automatic trigger here? If not, why is that?

File: ios_debug_proxy.yml [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+          unzip openssl.zip
+          unzip libplist.zip
+          unzip libusbmuxd.zip
+          unzip libimobiledevice.zip
+          unzip ios-webkit-debug-proxy.zip
+          unzip pcre.zip
+          pwd
+          ls -l
+###################
+# Installation of neccesary builds
```
Comment (AnaIsabelSousa) [inline ios_debug_proxy.yml]: Minor: neccesary -> necessary

File: pipelines/azure_pipeline_example.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
+#Comments
```
Comment (AnaIsabelSousa) [inline pipelines/azure_pipeline_example.yml]: Do we need this file? Or can it be deleted? It looks like an example only

File: pipelines/ios_debug_proxy.yml [hunk 5]
```diff
@@ -0,0 +1,124 @@
+#Comments
```
Comment (AnaIsabelSousa) [inline pipelines/ios_debug_proxy.yml]: What's the difference between this file and " ios_debug_proxy.yml"? From their name it seems they are serving the same purpose

File: ios_debug_proxy.yml (hunk 2 above)
Comment (scout719) [reply ios_debug_proxy.yml]: I don't know 😅 I need your help setting this up properly to run whenever there is a commit on outsystems

Comment (AnaIsabelSousa) [reply ios_debug_proxy.yml]: A commit in the PS or in here?
If it's here something like should do the trick
```
trigger:
- main
```

**Review by AnaIsabelSousa — APPROVED** [review]

**Pull request merged**